src/Adapter/Mysql.php

```diff
@@ -23,30 +23,30 @@ public function getOptions(Config $config)
     {
         $options = parent::getOptions($config);
 
-        if (isset($config->use_ssl) && $config->use_ssl === '1') {
-            if (isset($config->ssl_key)) {
+        if (isset($config->use_ssl) && $config->use_ssl) {
```

Comment:
And what happens here if `use_ssl = "0"`?

Reply:
`"0"` evaluates to false and `"1"` evaluates to true. I've removed the strict check to also take real boolean values into account. If we also want to account `"false"` and `"true"` we need to use `filter_var()`. Do you want that too?

Reply:
No, I just wondered why you're not using `empty` here as well.

Reply:
Yeah, that makes me wonder now too 😆